aggify/aggify.py

```diff
@@ -203,17 +203,15 @@ def __to_aggregate(self, query: dict[str, Any]) -> None:
                         if match != {}:
                             matches.append(match)
 
-                self.pipelines.extend(
-                    [
-                        self.__lookup(
-                            from_collection=from_collection._meta["collection"],  # noqa
-                            local_field=local_field,
-                            as_name=as_name,
-                        ),
-                        *[{"$match": match} for match in matches],
-                    ]
+                self.pipelines.append(
+                    self.__lookup(
+                        from_collection=from_collection._meta["collection"],  # noqa
+                        local_field=local_field,
+                        as_name=as_name,
+                    )
                 )
                 self.unwind(as_name)
+                self.pipelines.extend([{"$match": match} for match in matches])
```

Comment:
If only one thing is going to be added to the list, use append

Reply:
oh, I thought I've used extend, but it seems my fingers have written append 😂😂

Thanks 😁

Reply:
there may be more than one `$match` stage, so extending is necessary